src/background.ts

```diff
@@ -80,5 +80,18 @@ browser.storage.onChanged.addListener((changes, areaname) => {
 // Prevent Tridactyl from being updated while it is running in the hope of fixing #290
 browser.runtime.onUpdateAvailable.addListener(_ => {})
 
-// Try to load an RC file
-excmds.source_quiet()
+config.getAsync("autocmds", "TriStart").then(aucmds => {
```

Comment:
You can listen to an event for when the extension starts up with the browser. That would be more future proof for when we fix the hot updates.
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/onStartup